packages/host/src/api/hooks/sync/useSyncSettingsUpdate.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  ServerSettingsNode,
+  UpdateSyncSettingsInput,
+  useMutation,
+  useQueryClient,
+} from '@openmsupply-client/common';
+import { useHostApi } from '../utils/useHostApi';
+
+export const useSyncSettingsUpdate = () => {
+  const queryClient = useQueryClient();
+  const api = useHostApi();
+  return useMutation<
```

Comment:
explicitly setting the typing on `useMutation` calms typescript down a bit. Usually it's just variables and return type that I'm concerned with